packages/components/src/toolbar/README.md

```diff
@@ -1,5 +1,44 @@
 # Toolbar
 
+Toolbar can be used to group related options. To emphasize groups of related icon buttons, a toolbar should share a common container.
+
+![Toolbar component above an image block](https://wordpress.org/gutenberg/files/2019/01/s_96EC471FE9C9D91A996770229947AAB54A03351BDE98F444FD3C1BF0CED365EA_1541782974545_ButtonGroup.png)
+
+## Table of contents
+
+1. [Design guidelines](#design-guidelines)
+2. [Development guidelines](#development-guidelines)
+
+# Design guidelines
```

Comment:
Heading level issue

Reply:
Good catch! Just pushed b1e320c to fix that. Might giving it one more review? Thank you!